### PR TITLE
Decimal: adds missing test coverage

### PR DESCRIPTION
**Pull request by gangov**

### CODE REVIEW

File: packages/decimal/src/lib.rs [hunk 1]
```diff
@@ -861,4 +862,58 @@ mod tests {
     fn decimal_pow_overflow_panics() {
         _ = Decimal::MAX.pow(2u32);
     }
+
+    #[test]
+    fn decimal_abs_with_negative_number() {
+        let decimal = Decimal::new(-128);
+
+        assert_eq!(decimal.abs(), Decimal { 0: 128 });
+    }
```
Comment (ueco-jb) [inline packages/decimal/src/lib.rs:871]: Decimal should not have a negative number in this implementation.

File: packages/decimal/src/lib.rs [hunk 2]
```diff
@@ -861,4 +862,58 @@ mod tests {
...
+    fn decimal_displayed_as_string() {
+        let env = Env::default();
+        let decimal = Decimal::percent(128);
+        // Dmytro
+        // token.symbol().to_string().as_str(), "JOY"))
+        //
+        // Jakub
+        /// assert_eq!(a.to_string(&e), String::from_slice(&e, "1.234"));
+
+        assert_eq!(decimal.to_string(&env), String::from_slice(&env, "128"));
```
Comment (ueco-jb) [inline packages/decimal/src/lib.rs:891]: Remove comments.

**Review by ueco-jb — COMMENTED** [review]
The rest of tests is good. Good job.

File: packages/decimal/src/lib.rs (hunk 1 above)
Comment (gangov) [reply packages/decimal/src/lib.rs]: meaning the test is completely useless?

**Review by ueco-jb — APPROVED** [review]
LGTM

**Pull request merged**